applications/application_basic_usage.ipynb

```diff
@@ -0,0 +1,305 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "1d1a3a05",
+   "metadata": {},
+   "source": [
+    "The [Applications API](https://dev.writer.com/api-guides/applications) allows you to turn deployed [no-code applications](https://dev.writer.com/no-code/introduction) into microservices and tools.\n",
+    "\n",
+    "This cookbook shows how to use the Applications API to invoke deployed applications.\n",
+    "\n",
+    "## Prerequisites\n",
+    "\n",
+    "Before getting started, you'll need:\n",
+    "\n",
+    "- A [Writer AI Studio](https://app.writer.com/aistudio/signup?utm_campaign=devrel) account\n",
+    "- An API key, which you can obtain by following the [API Quickstart](https://dev.writer.com/api-guides/quickstart)\n",
+    "- A [no-code application deployed in Writer AI Studio](https://dev.writer.com/no-code/introduction)\n",
```

Comment:
Only text generation and research assistant applications work with this right now